cc-mt-service/src/routes/mtServiceRoutes.ts

```diff
@@ -0,0 +1,13 @@
+// Mt service routes
+import { Router, Request, Response } from "express";
+import {postAdvert} from '../handleData';
+
+const mtRouter = Router();
+
+mtRouter.post('/advert',async (req:Request, res:Response) => {
+    const response = await postAdvert();
```

Comment:
here, give the objectData to postAdvert() function. The object data should come from request body like in the openapi.yaml. You can access the request body attributes like this: req.body.attribute, for example req.body.title.